sdk/armcore/go.mod

```diff
@@ -3,6 +3,6 @@ module github.com/Azure/azure-sdk-for-go/sdk/armcore
 go 1.13
 
 require (
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.9.0
-	github.com/Azure/azure-sdk-for-go/sdk/internal v0.2.0
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.9.3
```

Comment:
Could you please bump this to 0.9.4 (released this morning).

Reply:
Done